code-challenges.test.js

```diff
@@ -51,12 +79,33 @@ const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deducti
 
 
 // b) Create the function that makes the test pass.
-
-
+// Pseudo Code:
+//Create a function with randomNouns as the name
+// the function is going to have arrays
+// Use the .join method to turn the array into a string
+// It's going to utilize the method .toUpperCase
+// Return with the .split to change the strings back to arrays
```

Comment:
Your pseudocode does not match the below function.  Please update your pseudocode to include all methods, functions, etc used inside your function.